src/pages/Dashboard/Profile.jsx

```diff
@@ -1,13 +1,294 @@
+                            value={user.bio}
+                            onChange={onChange}
+                        />
+                        {user.links.map(link => (
+                            <Input
+                                key={link.name}
+                                name={link.name}
+                                type="text"
+                                label={link.name}
+                                value={link.href}
+                                onChange={onChangeLinks}
+                            />
+                        ))}
+                        <ButtonContainer>
+                            <Button type="submit">Submit</Button>
+                        </ButtonContainer>
+                    </form>
+
+                    <Heading>Change Email</Heading>
+                    <form onSubmit={onSubmitEmailChange}>
+                        <Input
+                            name="email"
+                            type="text"
+                            label="Email"
+                            value={user.email}
+                            autoComplete="email"
+                            onChange={onChange}
+                        />
+                        <ButtonContainer>
+                            <Button type="submit">Submit</Button>
+                        </ButtonContainer>
+                    </form>
+
+                    <Heading>Change Password</Heading>
+                    <form onSubmit={onSubmitPasswordChange}>
+                        <Input
+                            name="currentPassword"
+                            type="password"
+                            label="Current Password"
+                            value={user.currentPassword}
+                            autoComplete="current-password"
+                            onChange={onChange}
+                            required
+                        />
+                        <Input
+                            name="newPassword"
+                            type="password"
+                            label="New Password"
+                            value={user.newPassword}
+                            autoComplete="new-password"
+                            onChange={onChange}
+                            required
+                        />
+                        <Input
+                            name="confirmPassword"
+                            type="password"
+                            label="Confirm New Password"
+                            value={user.confirmPassword}
+                            autoComplete="new-password"
+                            onChange={onChange}
+                            required
+                        />
+                        <ButtonContainer>
+                            <Button type="submit">Submit</Button>
+                        </ButtonContainer>
+                    </form>
+                </Column>
+                <Column>
```

Comment:
We can clean this component up a bit by moving it into another file as it's own component